zerver/lib/scim.py

```diff
@@ -31,6 +31,15 @@ class ZulipSCIMUser(SCIMUser):
 
     id_field = "id"
 
+    ROLE_TYPE_TO_NAME = {
+        UserProfile.ROLE_REALM_OWNER: "owner",
+        UserProfile.ROLE_REALM_ADMINISTRATOR: "administrator",
+        UserProfile.ROLE_MODERATOR: "moderator",
+        UserProfile.ROLE_MEMBER: "member",
+        UserProfile.ROLE_GUEST: "guest",
+    }
+    ROLE_NAME_TO_TYPE = {v: k for k, v in ROLE_TYPE_TO_NAME.items()}
```

Comment:
Do you think we should move to be an attribute on `UserProfile`, just so it's co-located with the definitions of these role values?  We're not likely to add additional roles in the future, but I kinda feel like it is still good practice.

Reply:
Hmm, my thinking was that we don't use these specific strings with such a capitalization anywhere else, so this is kinda a SCIM-specific map. We have this already:

https://github.com/zulip/zulip/blob/2ec288a9837eb1395d870d4ca05f9ed7eaf2f12d/zerver/models.py#L1976-L1982

but I don't think these verbose, nicely-capitalized names make sense to just use as the SCIM `role` values?

Reply:
Yeah, OK, that argument makes sense to me. We can always refactor later.